### ISSUE TITLE
Could you please make the book icon in GM tools optional?

### ISSUE THREAD

**Issue by TrofimSelbs** [caewok/fvtt-token-cover#21]
Hey, thank you for your module! I updated recently and was wondering about the functionality of the book tool in GM tool bar. Is there any way to make a toggle in settings to make this button optional? Thank you!

**Comment by caewok** [comment]
Yes, I will consider it. I borrowed it from my Terrain Mapper module, which borrowed it from DFred's Convenient Effects. Originally, I thought it should allow for drag-and-drop, but that actually seems rather unnecessary now. And it seems unlikely that a GM would fiddle with the settings during a game. So maybe it should be a button in the settings config instead.

**Comment by TrofimSelbs** [comment]
thank you!

**Comment by caewok** [comment]
Added in v0.7.3.
